.eslintrc.js

```diff
@@ -0,0 +1,222 @@
+const NodePlugin = require('./tools/node_modules/eslint-plugin-node');
+NodePlugin.RULES_DIR = 'tools/eslint-rules';
```

Comment:
Shouldn't this path be `path.join(__dirname, 'tools', 'eslint-rules')`, so that eslint can be called from subdirectories too?

Reply:
@silverwind i wasn't able to figure out what markdown was doing, it wasn't in tools/node_modules so i assumed it was builtin to eslint and didn't think about it any further

Reply:
Yes, it's in [`tools/node_modules/eslint/node_modules`](https://github.com/nodejs/node/tree/master/tools/node_modules/eslint/node_modules/eslint-plugin-markdown). I got the error about it missing when running my global `eslint` without it installed globally but if you don't see these errors, I think something might be wrong in my setup. I use yarn for global modules, if that matters.

Reply:
interesting, i have modified the script to fix this thanks 😄